cmd/thanos/compact.go

```diff
@@ -301,6 +302,9 @@ func runCompact(
 
 	ctx, cancel := context.WithCancel(context.Background())
 
+	// adding tracer to ctx
```

Comment:
Please use a complete sentence when you add comment.

Reply:
Yes, I'll do that in my upstream PR(remember this from the last 2 PRs :) )